doc/man/man8/daos.8

```diff
@@ -1,295 +1,976 @@
-.br
-	  \fB--value=\fRVALUESTR   container attribute value to set
-.TP
-.I container \fBOPTION\fRs (snapshot and rollback-related):
-	  \fB--snap=\fRNAME        container snapshot (create/destroy-snap, rollback)
-.br
-	  \fB--epc=\fREPOCHNUM     container epoch (destroy-snap, rollback)
-.br
-	  \fB--epcrange=\fRB-E     container epoch range (destroy-snap)
-.TP
-.I filesystem \fR(\fIfs\fR) \fBCOMMAND\fRs:
-	  \fBcopy\fR           	 copy data to/from a container to POSIX path
-.br
-	  \fBset-attr\fR     	 set attributes (object class or chunk size) of a (new) file or (existing directory)
-.br
-	  \fBget-attr\fR     	 retrieve attributes of a file or directory
-.br
-	  \fBreset-attr\fR     	 reset attributes to default of a directory
-.br
-	  \fBreset-chunk-size\fR reset chunk size to default of a directory
-.br
-	  \fBreset-oclass\fR   	 reset object class to default of a directory
-.TP
-.I filesystem \fR(\fIfs\fR) \fBOPTION\fRs:
-.br
-	  \fB--path=\fRPATHSTR        path of file or directory to access, including the UNS link to the DFS container
-.br
-	  \fB--chunk-size=\fRBYTES    chunk size of file or directory.
-.br
-	  \fB--oclass=\fROCLSSTR      container object class
-.br
-				      (ex: S{1,2,4,X}, RP_2G{1,2,4,X}, RP_3G{1,2,4,X}, RP_4G{1,2,4,X}, RP_XSF)
-.br
-	  \fB--dfs-path=\fRSTR        <path> used with \fB--cont\fR) and \fB--pool\fR) option to specify the path of the file or directory in the DFS container
-.br
-	  \fB--dfs-prefix=\fRSTR      <path> a prefix to be trimmed from the absolute path when looking up the dfs-path in the container
-.br
-	  \fB--src=\fRSTR             src=daos://<pool/cont>[/<path>] | <path>
-.br
-	  \fB--dst=\fRSTR             dst=daos://<pool/cont>[/<path>] | <path>
-.br
-		type is daos, only specified if pool/cont used
-.TP
-.I object \fR(\fIobj\fR) \fBCOMMAND\fRs:
-	  \fBquery\fR            query an object's layout
-.br
-	  \fBlist-keys\fR        list an object's keys
-.br
-	  \fBdump\fR             dump an object's contents
-.TP
-.I object \fR(\fIobj\fR) \fBOPTION\fRs:
-	  <\fIpool\fR options>   (\fB--pool\fR, \fB--sys-name\fR)
-.br
-	  <\fIcont\fR options>   (\fB--cont\fR)
-.br
-	  \fB--oid=\fRHI.LO        object ID
-
-.SH COLOPHON
-This page is part of the Distributed Asynchronous Object Storage (DAOS)
-exascale storage project.
-.br
-The project git reposity
-can be found at:
-\%https://github.com/daos-stack/daos.git .
-.br
-Bugs can be reported at:
-\%https://jira.hpdd.intel.com/projects/DAOS/.
+daos is a tool that can be used to manage/query pool content,
+create/query/manage/destroy a container inside a pool, copy data
+between a POSIX container and a POSIX filesystem, clone a DAOS container,
+or query/manage an object inside a container.
+.SH OPTIONS
+.SS Application Options
+.TP
+\fB\fB\-\-debug\fR\fP
+enable debug output
+.TP
+\fB\fB\-\-verbose\fR\fP
+enable verbose output (when applicable)
+.TP
+\fB\fB\-\-json\fR\fP
+enable JSON output
```

Comment:
This is great, there's presumably an amount of test code that could be updated to use this, although I'm not suggesting that we delay this PR any further.